spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/procedures/FastForwardBranchProcedure.java

```diff
@@ -73,19 +71,16 @@ public StructType outputType() {
   @Override
   public InternalRow[] call(InternalRow args) {
     Identifier tableIdent = toIdentifier(args.getString(0), PARAMETERS[0].name());
-    String source = args.getString(1);
-    String target = args.getString(2);
+    String from = args.getString(1);
+    String to = args.getString(2);
```

Comment:
Probably API and Spark should be separate PRs but since the Spark changes were small, I kept it in the same PR. If it makes it easier for reviewers I can separate.